src/applications/personalization/profile/tests/e2e/contact-information/focus-management.cypress.spec.js

```diff
@@ -51,39 +46,42 @@ describe('Contact info fields', () => {
       'be.focused',
     );
     cy.axeCheck();
-    cy.findByRole('button', { name: /cancel/i }).click();
+    cy.get('va-button[text="Cancel"]').click();
     cy.findByRole('button', { name: /edit home address/i }).should(
       'be.focused',
     );
 
     cy.findByRole('button', { name: /edit home.*number/i }).click();
```

Comment:
same question for this line regarding updating it?

Reply:
I think the label is "Edit home phone number" (the `.*` is RegExp that matches anything), but I'm not 100% sure so I'd rather not touch it.